### PR TITLE
Fall back to contra-hdsl

### LINKED ISSUE #1167: Revert back to contra-hdsl
with recent discussion, we need to revert back to contra-hdsl for running linchpin unittests

### PR DESCRIPTION
**Pull request by samvarankashyap**
According to the recent discussion with @maneeshmehra 
It was decided that we continue to use contra-hdsl for unittests until a better alternative comes by. 
It was also mentioned that contra-hdsl will be supported for existing customers.
fixes #1167

### CODE REVIEW

**Comment by samvarankashyap** [comment]
[test]

**Comment by samvarankashyap** [comment]
@p3ck It's failing due because the JenkinsUnittestfile is not found which was deleted in the first place. 
Is there any way we could make it run without merging this PR?

**Comment by p3ck** [comment]
[test]

File: playbooks/tests/unit_tests.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  gather_facts: False
+  tasks:
+          - name: debug
+            debug:
+              msg: "Disabling linchpin tests until container is fixed. Refer Travis ci for unit tests and flake8 tests"
+          - name: Install Test Reqs
+            pip:
+              name: .[tests]
+              virtualenv: /tmp/linchpin
+              args:
```
Comment (p3ck) [inline playbooks/tests/unit_tests.yml]: how did you restore this commit??  did you revert?  line 13 doesn't look like it's indented correctly

Comment (samvarankashyap) [reply playbooks/tests/unit_tests.yml]: It was commented before the unit-tests were completely removed. I uncommented them to start tests again.
Indentation went off while I uncommented the parts of the playbook

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**